libraries/lib-audio-devices/AudioIOBase.h

```diff
@@ -245,7 +245,7 @@ class AUDIO_DEVICES_API AudioIOBase /* not final */
    std::weak_ptr<AudacityProject> mOwningProject;
 
    /// True if audio playback is paused
-   bool                mPaused;
+   std::atomic<bool>   mPaused{ false };
```

Comment:
Compiling fails on windows because of missing #include <atomic> in this header. I added it (only locally) and could move on testing.

Reply:
Fixed in the rebased branch, no other changes